### DISCUSSION TITLE
Return error codes from API

### DISCUSSION THREAD

**Discussion by denolfe** [payloadcms/payload#1504] [Feature Requests & Ideas]
As of now, any handling of errors on the frontend has to be done via HTTP status code or error message. The introduction of i18n makes using error messages much more difficult. Having a set list of error codes maintained internally would make handling these much easier.

Ideally, in addition to the error message, we'd send back a field like `errorCode` which would be standardized.

**Comment by dolanoadriano** [comment]
ha, how's it going? :D
